pom.xml

```diff
@@ -124,6 +123,14 @@
 			<artifactId>javabean-tester</artifactId>
 			<version>1.0.0</version>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/me.ramswaroop.jbot/jbot -->
+		<dependency>
```

Comment:
I believe this is still an extra dependency since you have the 4.1.0 also listed

Reply:
fixed now, thanks!